app/views/fragments/metricsHeader.scala.html

```diff
@@ -0,0 +1,23 @@
+@(oldInstancesCount: Int, totalInstancesCount: Int, agePercentiles: utils.Percentiles)
+
+@percentileRow(percentileKey: Int, percentileValue: Int) = {
```

Comment:
I see fragments as reusable pieces of template. Here a `percentileRow` has no meaning outside of this template which defined the rows container. It is only a code block used to avoid duplication.